lib/components/map/connected-geojson-layer.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  Styled as BaseMapStyled,
+  MarkerWithPopup
+} from '@opentripplanner/base-map'
+import { connect } from 'react-redux'
+import FromToLocationPicker from '@opentripplanner/from-to-location-picker'
+import React, { useEffect, useState } from 'react'
+
+import { setLocation } from '../../actions/map'
+import { SetLocationHandler } from '../util/types'
+
+type Props = {
+  setLocation: SetLocationHandler
+  url: string
+}
+const GeoJSONOverlay = (props: Props) => {
+  const { setLocation, url } = props
+
+  const [locations, setLocations] = useState([])
+  useEffect(() => {
+    async function downloadLocations() {
+      const json = await (await fetch(url)).json()
+      setLocations(json)
+    }
+    if (url) downloadLocations()
+  }, [url])
+
+  return (
+    <>
+      {/* @ts-expect-error TODO: geojson types */}
```

Comment:
Try setting the type of `locations` to `GeoJSON.FeatureCollection` (do it where you call `useState`).